Population.cpp

```diff
@@ -830,38 +839,21 @@ void Population::recruitMany(std::vector<Individual*>& new_inds) {
 
 //---------------------------------------------------------------------------
 
-// Transfer is run for populations in the matrix only
-#if RS_RCPP // included also SEASONAL
-int Population::transfer(Landscape* pLandscape, short landIx, short nextseason)
-#else
-int Population::transfer(Landscape* pLandscape, short landIx)
-#endif
+// Transfer is run for a given vector of individuals
+int Population::transfer_move(Species* pSpecies, std::vector<Individual*>& inds, Landscape* pLandscape, short landIx)
```

Comment:
```suggestion
int Population::transfer_move(Species* pSpecies, std::vector<Individual*>& dispInds, Landscape* pLandscape, short landIx)
```
name `inds` is already used by the member variable of `Population`

Reply:
Could you simply confirm (by accepting the proposed changes) that all instances of calls to `inds` in these functions are meant to affect the argument, not the member variable?